backend/lib/endpoints/geo.js

```diff
@@ -7,7 +7,7 @@ const { getCountrySchema } = require("./schema/geo");
 async function routes(app) {
   app.get(
     "/country",
-    { preValidation: [app.authenticate], schema: getCountrySchema },
+    { schema: getCountrySchema },
```

Comment:
why remove this auth check? well not sure how this API call is used at the moment, we might need to add some additional security checks later

Reply:
The only place I see this being called is when going through the "Need Help" flow to get to the page where it provides an emergency number, so I'm not sure if authentication is necessary to simply get the geolocation from a set of lat/long coordinates.

Reply:
yeah you're right I think for now this is mostly used by unauthenticated users
we could replace it by external services later and we can also consider it for the create profile page and similar to enter the location data, might only get geo coordinates and automatically fill some other fields to make sure it's correctly set